mne_bids/mne_bids.py

```diff
@@ -511,8 +506,8 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         Acquisition parameter for the dataset.
     run : int | None
         The run number for this dataset.
-    kind : str, one of ('meg', 'eeg', 'ieeg')
-        The kind of data being converted. Defaults to "meg".
+    kind : str, one of ('auto', 'meg', 'eeg', 'ieeg')
+        The kind of data being converted. Defaults to "auto".
```

Comment:
Maybe mention that `kind = 'auto'` will determine the type of data automatically from the input raw data? Or is it obvious enough from the name?